# Triterpenoid production in the LECA #
Data for preprint [Francis, W.R. (XXXX) The eukaryotic last common ancestor was bifunctional for hopanoid and sterol production](https://www.preprints.org/manuscript/202004.0186)

### Abstract ###
Steroid and hopanoid biomarkers can be found in ancient rocks and, in principle, can give a glimpse of what life was present at that time. Sterols and hopanoids are produced by two related enzymes, though the evolutionary history of this protein family is complicated by losses and horizontal gene transfers, and appears to be widely misinterpretted. Here, I have added sequences from additional key species, and re-analysis of the phylogeny of SHC and OSC indicates a single origin of both enzymes among eukaryotes. This pattern is best explained by vertical inheritance of both enzymes from a bacterial ancestor, followed by widespread loss of SHC, and two subsequent HGT events to ferns and ascomycetes. Thus, the last common ancestor of eukaryotes would have been bifunctional for both sterol and hopanoid production. Later enzymatic innovations allowed diversification of sterols in eukaryotes. Contrary to previous interpretations, the LCA of eukaryotes potentially would have been able to produce hopanoids as a substitute for sterols in anaerobic conditions. Without invoking any other metabolic demand, the LCA of eukaryotes could have been a facultative aerobe, living in unstable conditions with respect to oxygen level.

### Supplemental data ###
* [full dataset in fasta format](https://bitbucket.org/wrf/euk-sterol-hopene/downloads/erg7_proteins_preprint_v1.fasta.gz)
* [alignment and iqtree results](https://bitbucket.org/wrf/euk-sterol-hopene/downloads/iqtree_full_alignment.zip)
* [trimmed alignment and iqtree](https://bitbucket.org/wrf/euk-sterol-hopene/downloads/iqtree_trimmed_alignment.zip)

### Datasets ###
* Initial dataset was from [Takishita 2012](https://www.doi.org/10.1186/1745-6150-7-5), received [from the authors](https://bitbucket.org/wrf/euk-sterol-hopene/downloads/ERG7_from_takishita2012.fasta.gz)
* Sequences were added from [Gold 2017](https://www.doi.org/10.1038/nature21412), based on accessions
* Bacterial sequences were mostly from NCBI, after multiple rounds of blast with many queries
* Single-celled eukaryotic sequences were mostly from [MMETSP](https://github.com/wrf/misc-analyses/tree/master/marine_meta), by [Keeling 2014](https://www.doi.org/10.1371/journal.pbio.1001889)

### Misc ###
This project is supported by the VILLUM Experiment grant (no. 00028022) to WRF.
